README.adoc

```diff
@@ -107,154 +92,157 @@ mvn liberty:stop-server
-[source, java, linenums, role='code_column']
+[source, java, linenums, role='code_column hide_tags=copyright tags=InventoryLivenessCheck']
 ----
-include::finish/src/main/java/io/openliberty/guides/inventory/InventoryLivenessCheck.java[tags=InventoryLivenessCheck]
+include::finish/src/main/java/io/openliberty/guides/inventory/InventoryLivenessCheck.java[]
 ----
 
-As with the `system` liveness check, you are checking the heap memory usage. If more 
-than 90% of the maximum memory is being used, a `DOWN` status is returned.  
+As with the `system` liveness check, you are checking the heap memory usage. If more
+than 90% of the maximum memory is being used, a `DOWN` status is returned.
 
 CustomConfigSource.json
 [source, json, linenums, role='code_column']
 ----
-include::finish/resources/CustomConfigSource.json[tags=**]
+include::finish/resources/CustomConfigSource.json[]
 ----
 
 // =================================================================================================
 // Building and running the application
 // =================================================================================================
 
 [role='command']
-include::{common-includes}/mvnbuild.adoc[]
+include::{common-includes}/devmode-build.adoc[]
 
 While the server is running, navigate to the http://localhost:9080/health[http://localhost:9080/health^] URL to find
 the aggregated liveness and readiness health reports on the two services.
 
 You can also navigate to the http://localhost:9080/health/ready[http://localhost:9080/health/ready^]
-URL to view the readiness health report, or the http://localhost:9080/health/live[http://localhost:9080/health/live^] 
+URL to view the readiness health report, or the http://localhost:9080/health/live[http://localhost:9080/health/live^]
 URL to view the liveness health report.
 
 Put the `inventory` service in maintenance by setting the [hotspot=3]`io_openliberty_guides_inventory_inMaintenance`
-property to `true` in the [hotspot]`resources/CustomConfigSource.json` file. Because 
-this configuration file is picked up dynamically, simply refresh the http://localhost:9080/health[http://localhost:9080/health^] 
-URL you will see that the state of the `inventory` service has changed to `DOWN`. The 
-overall state of the application has also changed to `DOWN` as a result. Point to the 
-http://localhost:9080/inventory/systems[http://localhost:9080/inventory/systems^] URL to verify that the `inventory` service is 
+property to `true` in the [hotspot]`resources/CustomConfigSource.json` file. Because
+this configuration file is picked up dynamically, simply refresh the http://localhost:9080/health[http://localhost:9080/health^]
+URL you will see that the state of the `inventory` service has changed to `DOWN`. The
+overall state of the application has also changed to `DOWN` as a result. Point to the
+http://localhost:9080/inventory/systems[http://localhost:9080/inventory/systems^] URL to verify that the `inventory` service is
```

Comment:
Change
" Because this configuration file is picked up dynamically, simply refresh the http://localhost:9080/health[http://localhost:9080/health^] URL you will see that the state of the `inventory` service has changed to `DOWN`. The overall state of the application has also changed to `DOWN` as a result. Point to the http://localhost:9080/inventory/systems[http://localhost:9080/inventory/systems^] URL to verify that the `inventory` service is indeed in maintenance. Set the [hotspot=3]`io_openliberty_guides_inventory_inMaintenance` property back to `false` once you are done.
To
"Because this configuration file is picked up dynamically, simply refresh the http://localhost:9080/health[http://localhost:9080/health^] URL to see that the state of the `inventory` service changed to `DOWN`. The overall state of the application also changed to `DOWN` as a result. Go to the
http://localhost:9080/inventory/systems[http://localhost:9080/inventory/systems^] URL to verify that the `inventory` service is indeed in maintenance. Set the `io_openliberty_guides_inventory_inMaintenance` property back to `false` after you are done."

Reply:
Updated